aten/src/ATen/native/native_functions.yaml

```diff
@@ -13085,6 +13087,7 @@
   variants: function, method
   structured_delegate: isposinf.out
   dispatch:
+    NestedTensorCPU, NestedTensorCUDA: NestedTensor_isposinf
```

Comment:
@malfet We need to add test to `test_activations` in test_nested_tensor after the 2 week period has passed